DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -88,31 +80,47 @@ private static void runStreamHlsAndDash(ConfigWrapper config) {
 
         Scanner scanner = new Scanner(System.in);
```

Comment:
What does this Scanner do? Should probably comment this.

Reply:
Scanner is how this code waits for customer to enter input on the console.